<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentWidget</class>
 <widget class="QWidget" name="rentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rent estimator</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>751</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Area, m2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="areaEditBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>District</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="districtComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Голосіївський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дарницький</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Деснянський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дніпровський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Оболонський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Печерський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Подільський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Святошинський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Солом'янський</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Шевченківський</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Rooms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="roomComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Comfortness, %</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="comfortEditBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Time to a subway, minutes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="subwayTimeEditBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="estimateButton">
        <property name="text">
         <string>Estimate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="estimateResultLabel">
      <property name="text">
       <string>Estimation result:</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>all</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>very low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>below medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>above medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>high</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>extremely high</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="filterButton">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="flatsTableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
